File: src/main/java/com/recreations/service/CityService.java
package com.recreations.service;

import com.recreations.model.City;

import java.util.List;

public interface CityService {
    List<City> getAll();

    City get(int cityId);

    void add(City newCity);

    int remove(City city);

    void update(City city);
}

File: recreations-ui/src/services/CityService.js
import React from 'react'
import Service from './Service';

const ENDPOINT = 'city';

export default class CityService extends Service {
  getAllCities() {
    return super.getAll(ENDPOINT);
  }

  getCity(id) {
    return super.getById(ENDPOINT, id);
  }
}